css/style.css

```diff
@@ -12,6 +12,10 @@
 	box-shadow: 0 1px 0 rgba(0,0,0,.1) inset !important;
 }
 
+.nc-25 #body-login #password {
```

Comment:
Just make the min version 25?

Reply:
I wasn't sure since thiw app is currently supporting multiple versions, but that would indeed simplify things